content/blog/2019-08-13-popler.md

```diff
@@ -0,0 +1,81 @@
+
+The API also need be managed in order to avoid crashes which make downloads impossible. The API crashes whenever the online database is stopped and restarted, and whenever the API web service crashes. Stopping and re-starting the database occur, for example, when we want changes to the popler online database configuration to take effect. In those rare cases, the API will crash after stopping the database, and it will need to be restared, taking a few hours before being fully functional again. Moreover, the API will stop working whenever its web service crashes. These crashes occasionally happen, and they require restarting the web server of the API. To monitor whether the web service of our API is running, we use [Uptime Robot](https://uptimerobot.com/), a free application that quantifies the time during which the web server is available to users.
+
+### Conclusion: working with LTER population data
+
+Our work with popler highlighted many opportunities linked to data synthesis using population data from the USA LTER network. Specifically, we found that population time-series data, while containing sometimes daunting heterogeneities, also present conspicuous regularities. By "regularities", we mean that population abundance can be quantified in just a few ways (e.g. using densities, counts, area covered, ect.), that all datasets have a temporal replication of population censuses, a spatial replication structure (which is often times nested), and so on. Ecologists can use these regularities to format most population time-series using the same data model. `popler` suggests that the population data associated with the USA LTER data, and with other long-term population sampling efforts, provide untapped potential to foster synthesis work in population, community, and macro ecology.
+
+#### Acknowledgements
+
+We thank reviewers [Corinna Gries](https://github.com/cgries), and [Benjamin Bond-Lamberty](https://github.com/cgries) for helpful comments that improved the package functionality.
```

Comment:
fix link (cgries was duplicated for Benjamin): `[Benjamin Bond-Lamberty](https://github.com/bpbond)`